source/interface/azure_iot_transport_interface.h

```diff
@@ -0,0 +1,182 @@
+/* Copyright (c) Microsoft Corporation. All rights reserved. */
+/* SPDX-License-Identifier: MIT */
+
+/**
+ * @file azure_iot_transport_interface.h
+ * @brief Transport interface definitions to send and receive data over the
+ * network.
+ */
+
+#ifndef AZURE_IOT_TRANSPORT_INTERFACE_H
+#define AZURE_IOT_TRANSPORT_INTERFACE_H
+
+#include <stdint.h>
+#include <stddef.h>
+
+/**
+ * @transportpage
```

Comment:
Do we want this "@transportpage" since I don't think(?) we use this other places.

Reply:
agreed this is doxygen fluff which we don't use

Reply:
removed

---
In reply to: [598967801](https://github.com/Azure/azure-iot-middleware-freertos/pull/44#discussion_r598967801) [](ancestors = 598967801)